### PR TITLE
[test] add a new test for LLM class

### PR DESCRIPTION
**Pull request by skcoirz**
- added unit test cases for LLM base class.
- covering both realtime and async entry points.

### CODE REVIEW

**Comment by dev2049** [comment]
what the use case? don't we already have fake llms

**Comment by skcoirz** [comment]
> what the use case? don't we already have fake llms

this is just a simple one time class, being used in this test case only. The difference is the call function. This class here simply returns the prompt. Fake LLM class requires several more steps for a similar result. There is no major difference. Using this simple class here makes the test logics much simpler.

This test case is added for LLM Class generation and ageneration functions.

File: tests/unit_tests/llms/test_lllm_class.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+"""Fake LLM wrapper for testing purposes."""
```
Comment (hwchase17) [inline tests/unit_tests/llms/test_lllm_class.py:1]: typo in file name (lllm)

File: tests/unit_tests/llms/test_lllm_class.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+from langchain.callbacks.manager import (
+    AsyncCallbackManagerForLLMRun,
+    CallbackManagerForLLMRun,
+)
+from langchain.llms.base import LLM
+from langchain.schema import LLMResult
+
+
+class FakeNewSignLLM(LLM):
```
Comment (hwchase17) [inline tests/unit_tests/llms/test_lllm_class.py:14]: the whole point of a FakeLLM is to use in other tests right? at very least, if we think this is useful, we should put in the fake_llm file with the other fake llm

**Comment by baskaryan** [comment]
stale

**Pull request closed without merging**